torch/nn/quantized/modules/linear.py

```diff
@@ -123,11 +124,12 @@ class Linear(torch.nn.Module):
         >>> print(output.size())
         torch.Size([128, 30])
     """
-    _version = 3
+    _version = 4
```

Comment:
Why is the version number bumped up here?

Reply:
sorry, this is left over from previous changes, will revert